src/saltext/vmware/modules/esxi.py

```diff
@@ -9,6 +9,7 @@
 import saltext.vmware.utils.esxi as utils_esxi
 import saltext.vmware.utils.vmware as utils_vmware
 from salt.defaults import DEFAULT_TARGET_DELIM
+from salt.ext import six
```

Comment:
do we still need six given Py 3 been around a long time now, if Py 2-ism, surely that should be updated ?